src/openrct2/paint/sprite/Paint.Sprite.cpp

```diff
@@ -111,6 +112,12 @@ void sprite_paint_setup(paint_session* session, const uint16_t x, const uint16_t
         {
             case SPRITE_IDENTIFIER_VEHICLE:
                 vehicle_paint(session, (Vehicle*)spr, image_direction);
+#ifdef __ENABLE_LIGHTFX__
```

Comment:
The below functions should only be called on Head vehicles. The function is assuming it is only passed heads.

Reply:
On second thoughts that wouldn't work ignore the above comment.

Reply:
On third thoughts this is a mess and I am no longer sure what it is meant to do.

Reply:
(I'm marking this as resolved then Duncan)